README.md

```diff
@@ -1,24 +1,30 @@
-# **Node.js API (N-API) Package**
-
-This package contains **header-only C++ wrapper classes** for the 
-**ABI-stable Node.js API** also known as **N-API**, providing C++ object 
-model and exception handling semantics with low overhead. It guarantees 
-backward compatibility when used with older versions of Node.js that do 
-not have N-API built-in.
-
-N-API is an API for building native addons. It is independent from the 
-underlying JavaScript runtime (e.g. V8 or ChakraCore) and is maintained as 
-part of Node.js itself. This API will be Application Binary Interface (ABI) 
-stable across versions and flavors of Node.js. It is intended to insulate 
+# **node-addon-api module** 
+This module contains **header-only C++ wrapper classes** which simplify
+the use of the C based [N-API](https://nodejs.org/dist/latest/docs/api/n-api.html)
+provided by Node.js when using C++. It provides a C++ object model
+and exception handling semantics with low overhead.
+
+N-API is an ABI stale API for building native addons in C++. It is
```

Comment:
I think you just mean "N-API is a stable API" rather than "N-API is an ABI stale API"

Reply:
I think you should also say `N-API is a ABI stable C interface provided by Node.js for building native addons...` It is not necessary that such addons be written in C++ and saying `in C++` muddies the waters a little bit IMO.

Reply:
Thanks for catching those 2,  will update to address